stock_request_analytic/models/stock_request_order.py

```diff
@@ -1,81 +0,0 @@
-# Copyright 2018 Creu Blanca
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
-
-from odoo import _, api, fields, models
-from odoo.exceptions import ValidationError
-
-MAP_ACTIONS = {
-    "analytic_account": "analytic.action_account_analytic_account_form",
-    "analytic_tag": "analytic.account_analytic_tag_action",
-}
-MAP_FIELDS = {
-    "analytic_account": "analytic_account_ids",
-    "analytic_tag": "analytic_tag_ids",
-}
-MAP_VIEWS = {
-    "analytic_account": "analytic.view_account_analytic_account_form",
-    "analytic_tag": "analytic.account_analytic_tag_form_view",
-}
-
-
-class StockRequestOrder(models.Model):
-    _inherit = "stock.request.order"
-
-    analytic_count = fields.Integer(
-        compute="_compute_analytic_ids",
-        readonly=True,
-        compute_sudo=True,
-    )
-    analytic_tag_count = fields.Integer(
-        compute="_compute_analytic_ids",
-        readonly=True,
-        compute_sudo=True,
-    )
-    analytic_account_ids = fields.One2many(
-        comodel_name="account.analytic.account",
-        compute="_compute_analytic_ids",
-        string="Analytic Accounts",
-        readonly=True,
-        compute_sudo=True,
-    )
-    analytic_tag_ids = fields.One2many(
-        comodel_name="account.analytic.tag",
-        compute="_compute_analytic_ids",
-        string="Analytic Tags",
-        readonly=True,
-        compute_sudo=True,
-    )
-    default_analytic_account_id = fields.Many2one(
```

Comment:
A similar feature should be added

Reply:
What functionality exactly? The analytical accounts are no longer "useful", analytical distribution (from stock.request) is now what matters. Do you want to add a default analytical distribution at stock.request.order level?

Reply:
That the analytic distribution is filled with this default analytic account. Anyway, the same, check if it's used or not in our customers.